src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -321,6 +324,71 @@ def fuzz(self,
                                 artifact_prefix, additional_args, extra_env)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  SSH_RETRIES = 10
+  SSH_WAIT = 2
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
+        not environment.get_value('FUCHSIA_PORTNUM')):
+      raise fuchsia.errors.FuchsiaConfigError(
+          'FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
+    # yapf: disable
```

Comment:
use "#pylint: disable=" with the particular rule name than yapf.

Reply:
unfortunately this isn't a pylint rule as far as i can tell--the formatting is perfectly *fine* according to PEP, but yapf is overzealous and tries to fit as many list elements as possible onto a single line.  so really we want yapf to be less aggressive, rather than disabling any particular violation